automation/taskcluster/lib/tasks.py

```diff
@@ -327,22 +327,20 @@ def _craft_default_task_definition(
     def craft_master_commit_signing_task(
         self, assemble_task_id, variant
     ):
-        architecture, build_type, product = _get_architecture_and_build_type_and_product_from_variant(variant)
-        product = convert_camel_case_into_kebab_case(product)
-        postfix = convert_camel_case_into_kebab_case('{}-{}'.format(architecture, build_type))
+        architecture, build_type = _get_architecture_and_build_type_and_product_from_variant(variant)
         routes = [
             'index.project.mobile.fenix.branch.master.revision.{}.{}.{}'.format(
-                self.commit, product, postfix
+                self.commit, build_type, architecture
```

Comment:
Making this change makes sense. I wonder if we should use a new v2 scheme to avoid confusion. For instance:

```py
'index.project.mobile.fenix.v2.branch.master.revision.{}.{}.{}'
```

What do you think?

Reply:
Sure!